### ISSUE TITLE
Add Support for CocoaPods

### ISSUE THREAD

**Issue by mkalmes** [aaronabentheuer/AAFaceDetection#2]
I would love to integrate your code via CocoaPods. Could you please add support for CocoaPods with a [podspec](https://guides.cocoapods.org/making/making-a-cocoapod.html)?

**Comment by aaronabentheuer** [comment]
Yep. I will very soon publish a big update to AAFaceDetection and it will support CocoaPods.
